Traceback (most recent call last):
  File "/home/andrewb/GA_feature_selection/lasso.py", line 241, in <module>
    population, best_model_info = run_GA(executor, population, predictors, response_variables)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/andrewb/GA_feature_selection/lasso.py", line 169, in run_GA
    models = evaluate_fitness_parallel(executor, population, predictors, response_variables)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/andrewb/GA_feature_selection/lasso.py", line 114, in evaluate_fitness_parallel
    batch_results = future.result()
                    ^^^^^^^^^^^^^^^
  File "/sw/easybuild_milan/software/Anaconda3/2024.02-1/lib/python3.11/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/sw/easybuild_milan/software/Anaconda3/2024.02-1/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
concurrent.futures.process.BrokenProcessPool: A process in the process pool was terminated abruptly while the future was running or pending.
Script failed with exit code 1
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=552275.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
